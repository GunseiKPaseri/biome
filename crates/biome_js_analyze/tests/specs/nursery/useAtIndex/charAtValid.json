[
  "string.charAt(string.length - 0);",
  "string.charAt(string.length + 1)",
  "string.charAt(string.length + -1)",
  "foo.charAt(bar.length - 1)",
  "string?.charAt?.(string.length - 1);",
  "string?.charAt(string.length - 1);",
  "string.charAt(9);",
  "string1.charAt(string2.length - 1);",
  "string.charAt(hoge.string.length - 1)"
]
