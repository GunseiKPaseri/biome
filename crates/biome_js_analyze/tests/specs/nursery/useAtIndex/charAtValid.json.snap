---
source: crates/biome_js_analyze/tests/spec_tests.rs
assertion_line: 86
expression: charAtValid.json
---
# Input
```cjs
string.charAt(string.length - 0);
```

# Input
```cjs
string.charAt(string.length + 1)
```

# Input
```cjs
string.charAt(string.length + -1)
```

# Input
```cjs
foo.charAt(bar.length - 1)
```

# Input
```cjs
string?.charAt?.(string.length - 1);
```

# Input
```cjs
string?.charAt(string.length - 1);
```

# Input
```cjs
string.charAt(9);
```

# Input
```cjs
string1.charAt(string2.length - 1);
```

# Input
```cjs
string.charAt(hoge.string.length - 1)
```

# Input
```cjs
string.charAt(string.length - 1 + 1)
```

# Input
```cjs
string.charAt(string.length + 1 - 1)
```
